metapool/metapool.py

```diff
@@ -48,7 +48,8 @@ def read_plate_map_csv(f, sep='\t'):
 
 
 # method to read minipico output
-def read_pico_csv(f, sep='\t', conc_col_name='Sample DNA Concentration'):
+def read_pico_csv(f, sep='\t', plate_reader='Synergy_HT',
+    conc_col_name='Sample DNA Concentration'):
```

Comment:
This line needs to be aligned with the arguments in the previous line. If you run `flake8 metapool/metapool.py` you'll see the error I'm talking about.


This should fix it:
```suggestion
                  conc_col_name='Sample DNA Concentration'):
```

Reply:
sounds good, now I know how to check for pycodestyle.